### DISCUSSION TITLE
New Feature addition

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [WeMakeDevs/classroom-monitor-bot#2] [Ideas]
Suggest some new features you would like you monitor to have

**Comment by euhidaman** [comment]
Can we also add, more languages support, as even international students are part of the community. So, those ppl who have problems communicating in English can use it freely.

**Comment by utkarsh1311** [comment]
Can I add some more words to the list of edTech companies list in restrictedWords.js?

**Reply by kaiwalyakoparkar** [reply]
@utkarsh1311 Yes feel free to do that. Just make sure you create an issue first and tell me what words you are going to add. If that get's confirmed you can create a PR.
